src/applications/caregivers/config/chapters/primary/primaryContact.js

```diff
@@ -1,19 +1,19 @@
 import fullSchema from 'vets-json-schema/dist/10-10CG-schema.json';
-import { PrimaryCaregiverInfo } from 'applications/caregivers/components/AdditionalInfo/PrimaryCaregiverInfo';
+import confirmationEmailUI from 'platform/forms-system/src/js/definitions/confirmationEmail';
```

Comment:
If this is asking the Veteran to enter their email twice, then it's an [anti-pattern](https://dsva.slack.com/archives/C0NGDDXME/p1656524036994269?thread_ts=1621367250.006000&cid=C0NGDDXME).

Reply:
This has been brought up to the team and tabled for further research and redesign by the UX folks.